### DISCUSSION TITLE
欧皇彩主题解锁

### DISCUSSION THREAD

**Discussion by quannen** [Meloong-Git/PCL#947] [提问] [answered]
请问“今日人品”为100的概率有多大？

**Comment by LTCatt** [comment, marked as answer]
你猜

**Reply by quannen** [reply]
大草（bushi...

**Reply by LTDove** [reply]
大草（bushi...

**Reply by JOGIRAFFER** [reply]
大草

**Comment by ghost** [comment]
反正我是每天测人品时无意中解锁的

**Comment by IF-desu** [comment]
lm貌似没有把隐藏主题部分的源码开源

**Reply by WorldHim** [reply]
简介上已经写了。。。

**Comment by 212112111zwl** [comment]
我测得人品100的第二天赞助获得了快照版，结果第二天测得人品二十几

**Reply by quannen** [reply]
可以改日期啊

**Reply by 212112111zwl** [reply]
但我不会啊

**Reply by 212112111zwl** [reply]
我电脑小白一个

**Reply by WorldHim** [reply]
不会建议 bdfs。。。

**Comment by XiaoYuTO233** [comment]
建议改日期，这样就可以，一直试，直到出100

**Reply by miner233** [reply]
没必要吧
每天打开 PCL2 试试，总会有的

**Comment by huruichen380** [comment]
[ohh]
<img width="1957" height="1275" alt="屏幕截图 2026-03-29 184045" src="https://github.com/user-attachments/assets/16a4b5c7-49a8-436a-9fed-7f102e4c6dfa" />
